aether-kernel/aether/kernel/api/project_artefacts.py

```diff
@@ -178,6 +183,7 @@ def upsert_project_artefacts(
                 action=action,
                 name=mapping_name,  # use same name as mapping
                 input=raw_mapping.get('input', {}),
+                schema=raw_mapping.get('schema', {}),
```

Comment:
I see an enhancement here, but I would leave it for a future task and it's to derive the input from the schema and the other way around if one of them is missing :thinking: